string box_type
geometry_msgs/PoseStamped pose_stamped
